### PR TITLE
Added publish snippet after copy locale

### LINKED ISSUE #3079: Snippet copy-locale command does not copy the changes to live workspace
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | ^1.3
| Browser Version | none

#### Actual Behavior

The command `sulu:snippet:locale-copy` copies the content only in the default workspace.

#### Expected Behavior

Publish the snippet automatically.

#### Steps to Reproduce

Create a snippet in `de` and run the command with `sulu:snippet:locale-copy de en` and see in phpcr:shell that the content is not in the live workspace.

#### Possible Solutions

Call publish after copy content.

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #3079 
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR add the publish command for copied locale document.

### CODE REVIEW

File: src/Sulu/Bundle/SnippetBundle/Snippet/SnippetRepository.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ public function getSnippetsByUuids(array $uuids, $locale, $loadGhostContent = fa
      *
      * @throws \InvalidArgumentException
      *
-     * @return SnippetBridge[]
+     * @return object[]
```
Comment (danrot) [inline src/Sulu/Bundle/SnippetBundle/Snippet/SnippetRepository.php]: Don't we have a better type hint?

File: src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php [hunk 2]
```diff
@@ -148,6 +153,8 @@ private function copyDocument($srcLocale, $destLocale, StructureBehavior $docume
             $destLocale,
             Structure::TYPE_SNIPPET
         );
+        $destDocument = $this->documentManager->find($document->getUuid(), $destLocale);
+        $this->documentManager->publish($destDocument, $destLocale);
```
Comment (danrot) [inline src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php]: Maybe we should add a comment saying this is only necessary as long the snippets do not support publishing by themselves.

**Comment by danrot** [comment]
I think you should also call `DocumentManager::flush()` at some point, otherwise that won't work 😉

File: src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php (hunk 2 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php]: i could also use `SnippetDocument`

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**